plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/DeployJob.java

```diff
@@ -216,9 +220,16 @@ private CloudSdk getCloudSdk(Path credentialFile,
         .exitListener(processExitListener)
         .appCommandMetricsEnvironment(CloudToolsInfo.METRICS_NAME)
         .appCommandMetricsEnvironmentVersion(CloudToolsInfo.getToolsVersion())
-        .appCommandOutputFormat("json")
-        .build();
-    return cloudSdk;
+        .appCommandOutputFormat("json");
+
+    IJavaProject javaProject = JavaCore.create(project);
```

Comment:
What's the benefit or rational? `DeployJob` can figure this out by itself, and we have one less parameter a caller has to pass in.